medusa/post_processor.py

```diff
@@ -332,6 +332,57 @@ def _delete(self, file_path, associated_files=False):
                 # do the library update for synoindex
                 notifiers.synoindex_notifier.deleteFile(cur_file)
 
+    @staticmethod
+    def rename_associated_files(new_path, new_base_name, cur_file_path):
+        """Rename associated file using media basename.
+
+        :param new_path: destination path where we want the file to
+        :param new_base_name: the media base filename (no extension) to use during the action
+        :param cur_file_path: full path of the associated file
+        :return: renamed full file path
+        """
+        # remember if the extension changed
+        changed_extension = None
+        # file extension without leading dot
+        extension = helpers.get_extension(cur_file_path)
+        # initally set current extension as new extension
+        new_extension = extension
+
+        if is_subtitle(cur_file_path):
+            code = cur_file_path.rsplit('.', 2)[1].lower().replace('_', '-')
+            if from_code(code, unknown='') or from_ietf_code(code, unknown=''):
+                if code == 'pt-br':
```

Comment:
In fact it could already be removed, it's just there to ensure support with some software (Kodi? Plex?) that has/had problems with pt-br (not recognized) vs. pr-BR (yes, case sensitive...).